src/Kirel.Repositories/Interfaces/IKirelGenericEntityFrameworkRepository.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System.Linq.Expressions;
+using Kirel.Repositories.Sorts;
+
+namespace Kirel.Repositories.Interfaces;
+
+/// <summary>
+/// Interface for generic repository
+/// </summary>
+/// <typeparam name="TKey">Type of special entity key</typeparam>
+/// <typeparam name="TEntity">Type of entity</typeparam>
+public interface IKirelGenericEntityFrameworkRepository<in TKey, TEntity> where TEntity : IKeyEntity<TKey> where TKey :  IComparable, IComparable<TKey>, IEquatable<TKey>
```

Comment:
Please make each "where on new line". And sync "where TEntity...." with with  KirelGenericEntityFrameworkRepository, i.e. we need: class, ICreatedAtTrackedEntity, IKeyEntity<TKey>

Reply:
Fixed